mep/common/templates/common/widgets/checkbox_fieldset.html

```diff
@@ -0,0 +1,18 @@
+{% with id=widget.attrs.id name=widget.attrs.name %}
+    <fieldset class="facet">
+    <legend>{{ name|title }}</legend>
```

Comment:
Why are you forcing to lower case when setting the label on the `FacetChoiceField` and then title casing here? Is that a Django convention? 

Seems like it would be easier and more straightforward to display as is. I'm trying to think whether there are any places this will cause problems with the facets we currently have planned and I don't know of anywhere it will break, but seems like it merits a brief comment or docstring note somewhere. (Probably the field rather than the template.)

Reply:
This is a bit of a tap-dance around trying to make most of the html attributes lower cased for style sake. It shouldn't matter for the HTML, though, so I'm happy to simplify.

Reply:
I could make one in the business logic of the widget and pass it. That might be clearer. The issue is that Django assumes that `label` lives with `form.Field` and the logic to render the widget would just render the options. I thought @thatbudakguy 's suggestion of having it in the widget was sensible here.

Reply:
I can see situations when the `<legend>` would (and should) be different from the `name=` attribute on the `<input>s`; like if the name of the field was multiple words long. seems like they should be separate values.